package.json

```diff
@@ -15,7 +15,11 @@
   "engines": {
     "node": ">= 20.0.0 <21"
   },
+  "dependencies": {
+    "typescript-eslint": "^8.8.1"
+  },
```

Comment:
Can this dependency be a `devDependency` instead? Or go into the `eslint-plugin` package? Or does it need to go in the same package.json as typescript?